db/db_settings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "dbconfig" : {
+      "username": "user",
+      "password": "*******",
```

Comment:
The password is actually configured somewhere else, I hope.

Not necessary for merging the PR but please discuss with the group how you plan to manage different environments (dev, testing, staging, production) and configurations for each. If you're not sure, get some suggestions from the gang! 😃